packages/frontend/src/features/omnibar/components/OmnibarFilters.tsx

```diff
@@ -51,8 +54,19 @@ type Props = {
     onSearchFilterChange: (searchFilters?: SearchFilters) => void;
 };
 
+function findUserName(
+    userUuid: string,
+    userList: OrganizationMemberProfile[] = [],
+) {
+    const user = userList.find((u) => u.userUuid === userUuid);
+
+    return user ? `${user.firstName} ${user.lastName}` : '';
```

Comment:
I think it will be better if you return undefined here

Reply:
done here: https://github.com/lightdash/lightdash/pull/9090/commits/958d69131a909d863d2eb1fb7c6bae6e7c601766